lib/Boolean.js

```diff
@@ -70,7 +77,7 @@
     Boolean.prototype.drawBackground = function(ctx, dimentions) {
       var statusWidth;
       Boolean.__super__.drawBackground.call(this, ctx, dimentions);
-      ctx.fillStyle = this.getStatus() ? config.get('color-success') : config.get('color-failure');
+      ctx.fillStyle = this.getStatus() == null ? config.get('color-unknown') : (this.getStatus() ? config.get('color-success') : config.get('color-failure'));
```

Comment:
Thank you for your feedback, I was not expecting this since this project seems quite old, it's cool to see you're still here !

> I'm not a fan of nested ternary conditions. This would look better expanded in an actual if/else-if/else.

I totally understand, and made the proper modifications. Feel free to tell me if there is anything else you'd change !